modules/security-osp-validating-certificates.adoc

```diff
@@ -0,0 +1,113 @@
+// This is included in the following assemblies:
+//
+// * installing/installing_openstack/preparing-to-install-on-openstack.adoc
+
+:_content-type: ASSEMBLY
+[id="security-osp-validating-certificates_{context}"]
+= Scanning {rh-openstack} endpoints for legacy HTTPS certificates
+
+Beginning with {product-title} 4.10, HTTPS certificates must contain subject alternative name (SAN) fields. Run the following script to scan each HTTPS endpoint in a {rh-openstack-first} catalog for legacy certificates that only contain the `CommonName` field.
+
+[IMPORTANT]
+{product-title} does not check the underlying {rh-openstack} infrastructure for legacy certificates prior to installation or updates. Use the provided script to check for these certificates yourself. Failing to update legacy certificates prior to installing or updating a cluster will result in cluster dysfunction.
+
+.Prerequisites
+
+* On the machine where you run the script, have the following software:
+** Bash version 4.0 or greater
+** grep
```

Comment:
Maybe markup `grep` tho I don't see it uniformly done in the repo so far. :shrug:

Reply:
```suggestion
** `grep`
```